### PR TITLE
Feat/redesign header controls

### PR DESCRIPTION
**Pull request by user1736**
sneak peek
image

### CODE REVIEW

File: WanderLost/WanderLost/Client/Components/NotificationToggle.razor.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+.notification-toggle__label {
+  display: block;
+  position: relative;
+
+  width: 42px;
+  height: 22px;
+
+  cursor: pointer;
+  text-indent: -9999px;
```
Comment (oleksandr-didyk) [inline WanderLost/WanderLost/Client/Components/NotificationToggle.razor.css:27]: ?

File: WanderLost/WanderLost/Client/Shared/NavMenu.razor [hunk 2]
```diff
@@ -23,19 +23,19 @@
     </svg>
   </NavLink>
   <div class="m-header__items">
-    <NavLink href="/Notifications" class="m-header__item" Match="NavLinkMatch.All">Notifications</NavLink>
-    <NavLink href="/PushNotifications" class="m-header__item" Match="NavLinkMatch.All">Push Notifications</NavLink>
-    <NavLink href="/About" class="m-header__item" Match="NavLinkMatch.All">About</NavLink>
+    <NavLink href="/Notifications" class="m-header__item brand-label" Match="NavLinkMatch.All">Notifications</NavLink>
```
Comment (oleksandr-didyk) [inline WanderLost/WanderLost/Client/Shared/NavMenu.razor:26]: Shouldn't `class` go before `href`?

**Review by oleksandr-didyk — APPROVED** [review]

File: WanderLost/WanderLost/Client/Components/NotificationToggle.razor.css (hunk 1 above)
Comment (user1736) [reply WanderLost/WanderLost/Client/Components/NotificationToggle.razor.css:27]: A way to hide label text so it's still accessible via screen readers. Not really needed here, but I decided to keep it anyway

File: WanderLost/WanderLost/Client/Shared/NavMenu.razor (hunk 2 above)
Comment (user1736) [reply WanderLost/WanderLost/Client/Shared/NavMenu.razor:26]: this way it's easier to see the pages at a glance

**Pull request merged**